_posts/2016-10-31-recurse-center-week-5.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: "Week[4] @ The Recurse Center 😳"
+comments: true
+categories: tech recurse
+---
+
+Almost half way through. Things getting serious!
```

Comment:
Bad grammar is part of the writing style, but yes, that would probably be better :)